modules/images/image-loading-optimization/optimization.php

```diff
@@ -0,0 +1,174 @@
+<?php
+/**
+ * Optimizing for image loading optimization.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+require_once __DIR__ . '/class-ilo-html-tag-processor.php';
```

Comment:
Similar to my comment on the other PR, can you put this into `load.php` to have all file requirements in one place?

Reply:
Done in 83d03621